test/functional/prioritise_transaction.py

```diff
@@ -109,7 +109,7 @@ def run_test(self):
         # This is a less than 1000-byte transaction, so just set the fee
         # to be the minimum for a 1000 byte transaction and check that it is
         # accepted.
-        self.nodes[0].prioritisetransaction(tx_id, int(self.relayfee*COIN))
+        self.nodes[0].prioritisetransaction(txid=tx_id, fee_delta=int(self.relayfee*COIN))
```

Comment:
There's another prioritisetransaction call below that needs to be updated (it's causing travis tests to fail).